backend/src/nodes/image_iterator_nodes.py

```diff
@@ -108,7 +108,7 @@ def walk_error_handler(exception_instance):
             if context.executor.should_stop_running():
                 return
 
-            for name in files:
+            for name in sorted(files):
```

Comment:
Out of curiosity: what comparison function does this use?

Reply:
Not sure. I can't find any documentation about the actual function it uses, but I've found posts saying it's O(nlogn) time complexity

Reply:
Aha! It uses timsort

Reply:
That's not what I meant. I want to know the comparison functions. There are a bunch of ways in which you can compare strings (e.g. by byte code, code point, character, etc.). I couldn't find any docs on that, so I thought you might know.

Reply:
> [The sort algorithm uses only < comparisons between items.](https://docs.python.org/3/library/functions.html#sorted)
> [Lexicographical ordering for strings uses the Unicode code point number to order individual characters.](https://stackoverflow.com/questions/4806911/how-are-strings-compared)

Found it. Python's sorting uses the elements `<` like C++ (even in the same duck-typing manner).